jdaviz/configs/imviz/plugins/rotate_image/rotate_image.py

```diff
@@ -0,0 +1,35 @@
+from matplotlib.transforms import Affine2D
+from traitlets import Any
+
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import TemplateMixin, ViewerSelectMixin
+
+
+@tray_registry('imviz-rotate-image', label="Simple Image Rotation")
+class RotateImageSimple(TemplateMixin, ViewerSelectMixin):
+    template_file = __file__, "rotate_image.vue"
+
+    angle = Any(0).tag(sync=True)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._theta = 0  # degrees, clockwise
+
+    def vue_rotate_image(self, *args, **kwargs):
+        # We only grab the value here to avoid constantly updating as
+        # user is still entering or updating the value.
+        try:
+            self._theta = float(self.angle)
+        except Exception:
+            return
+
+        viewer = self.app._viewer_by_id(self.viewer_selected)
+
+        # TODO: Is this really clockwise? Does it take negative angle?

```

Comment:
1. No, counterclockwise, as is/should be the convention for all ROI/region/subset rotations. Can add something to that effect to a pretransform docstring, but the actual docs are yet to be written.
2. Yes

Reply:
Huh... But I think `regions` theta is clockwise. Should glue canvas rotation be consistent with regions ? It would be super confusing if people rotate the *Subset* clockwise but the *canvas* counter-clockwise.

Reply:
No, in `regions` `angle` is documented as "anti-clockwise", and astropy/regions#390 also sets the matplotlib `*Selector.rotation` identical to `self.angle`. Fortunately, since otherwise also the glue-astronomy translator would have got it wrong. ;-)

Reply:
Thanks for the clarifications! I have corrected the angle direction.